src/rs/src/text.rs

```diff
@@ -215,18 +215,18 @@ impl<'a> SkillThing<'a> {
 				s.push_str("|1 Phantom");
 				Cow::from(s)
 			}
-			Skill::bubbleclear => Cow::from("Remove statuses (positive & negative) from target creature or permanent, & heal target creature 1.\nTarget gains a bubble. Bubbles nullify the next spell, ability, or spell damage used by opponent that targets or damages affected card"),
+			Skill::bubbleclear => Cow::from("Strip target creature or permanent of all statuses. If target is a creature, heal it by 1.\nTarget gains Bubble, protecting it from 1 spell or ability"),
 			Skill::butterfly => Cow::from(if self.set() == CardSet::Open {
-				"Target creature or weapon with either strength or HP less than 3 has its skills replaced with \"3:1 Destroy target permanent.\""
+				"Target creature or weapon with strength or HP less than 3 has its skills replaced with \"3:1 Destroy target permanent.\""
 			} else {
-				"Target creature with less attack than 3. Replace target's skills with \"3:1 Destroy target permanent\""
+				"Target creature with less attack than 3 has its skills replaced with \"3:1 Destroy target permanent\""
 			}),
 			Skill::burrow => Cow::from(if self.get_flag(Flag::burrowed) {
-				"Unburrow"
+				"Unburrow this creatue, doubling its strength"
 			} else {
 				"Burrow this creature. Strength is halved while burrowed"
 			}),
-			Skill::catapult => Cow::from("Sacrifice target creature you control to damage opponent for 100 * Creature's HP / (100 + Creature's HP). Frozen creautres deal 1.5x more. Poisoned creatures transfer their poison to opponent"),
+			Skill::catapult => Cow::from("Target a creature to sacrifice. Damage opponent based on target creature's HP†. Frozen creatures deal 1.5x damage. Poisoned creatures transfer poisons to opponent.\n\n† (100 * Creature's HP) / (100 + Creature's HP)"),
```

Comment:
```suggestion
			Skill::catapult => Cow::from("Sacrifice your own target creature. Damage opponent based on target creature's HP†. Frozen creatures deal 50% more damage. Poisoned creatures transfer poisons to opponent.\n\n† (100 * Creature's HP) / (100 + Creature's HP)"),
```

Reply:
Want to use unicode 50% like you did on main page, or keep card text as raw ascii?

Reply:
Prefer 50% here rather than unicode 1/2 since "deal one half more damage" reads weird